### DISCUSSION TITLE
Whatsapp Auth

### DISCUSSION THREAD

**Discussion by diegoalzate** [supabase/supabase#5620] [Questions] [answered]
If i have whatsapp sandbox configured in my twilio account, can i send whatsapp auth notification through the GoTrue Client ?

**Comment by ChinKX** [comment, marked as answer]
Hi, I am not sure if this is what you want but recently, the Supabase team just added a new feature for Whatsapp OTP login. Hope this will help!

![image](https://github.com/supabase/supabase/assets/40538074/4896d4ac-0e45-4bb2-940f-08c1b144d5ba)

For more details, please visit https://supabase.com/docs/guides/auth/phone-login/twilio
